client/src/nl/sensorlab/videowall/animation/baseanimations/SortingAnimation.java

```diff
@@ -0,0 +1,105 @@
+package nl.sensorlab.videowall.animation.baseanimations;
+import java.util.ArrayList;
+
+import com.cleverfranke.util.PColor;
+
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.BozoSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.BubbleSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.CocktailSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.GnomeSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.OddEvenSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.OptimizedBubbleSort;
+import nl.sensorlab.videowall.animation.baseanimations.sorting.Sort;
+import processing.core.PApplet;
+import processing.core.PGraphics;
+
+/**
+ * Visualization of different sorting methods
+ */
+public class SortingAnimation extends BaseAnimation {
+
+	private ArrayList<Sort> sortingmethods;
+	private int[] sortingColors = {}; // <-- Different colors for each sorting methods (just to make it more clear)
```

Comment:
I;d keep all the colors with the Sort class, which is also where the draw method is anyway.